v2/cmd/config/cloudrun.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+import (
+	"os"
+)
+
+func IsCloudRun() bool {
+	return os.Getenv("IS_CLOUDRUN") == "true"
```

Comment:
could we pass this as a key/value pair in the `config.toml` file?
`is_cloudrun = true`

so, we'd be extracting this value with `viper.GetBool`.  If the value is not there, the default in viper will be false. Asking because it seems that other config is usually read from the config file, so not sure why we're extracting it from the ENV var. 

Example: https://github.com/grafeas/voucher/blob/main/v2/cmd/voucher_server/server.go#L23

Reply:
oh, I see that you're using it here like so https://github.com/Shopify/voucher-internal/pull/154

ignore me, then

Reply:
We could, but we would need 2 configs for GKE and CloudRun. Would it be a better option instead of specifying env var once?

Reply:
no! this seems to be a reasonable approach as we need to keep both versions running